airflow/www/views.py

```diff
@@ -2288,6 +2292,53 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_repo = git_hash = None
+        try:
+            p = subprocess.Popen('pip freeze | grep airflow',
```

Comment:
Pip is the recommended way to install python packages as it lets you easily rollback, not to mention its the first step all of us took as per the folowing link http://pythonhosted.org/airflow/start.html

Also, this is a soft dependency. If PIP is not available, it will fall through to the git implementation. 

After a while, most people decided to run off their own forks. This is partly due to the unpredictably long release cycles. For those folks, they will install from their own git repos.